pages/common/asar.md

```diff
@@ -0,0 +1,19 @@
+# asar
+
+> A file archiver for the Electron platform.
```

Comment:
I know this got merged already, but what's the `Electron platform` in this instance, and how does that relate to file archiving? The only _Electron_ I'm aware of is the shell that the _Atom_ editor uses.

I'm confused.

Reply:
It's the same thing and its standard archive format.

https://github.com/electron/asar

I don't know if it's worth mentioning that's made for Electron. To me it seems more useful than the official description „Simple extensive tar-like archive format with indexing”, but that's up to you guys :)

Reply:
Opened #2325.

This description does seem more useful, I'm just curious as to whether we can clarify it a bit more :-)